core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -114,7 +114,8 @@ trait EntityComponent {
         val entityRecs = entities.toSeq.map(e => marshalNewEntity(e, workspaceContext.workspaceId))
 
         workspaceQuery.updateLastModified(workspaceContext.workspaceId) andThen
-          insertInBatches(entityQueryWithInlineAttributes, entityRecs) andThen entityQuery.getEntityRecords(workspaceContext.workspaceId, entityRecs.map(_.toReference).toSet)
+          DBIO.sequence(entityRecs.grouped(batchSize).map(entityQueryWithInlineAttributes ++= _)).map(_.flatten.sum).andThen(
+            entityQuery.getEntityRecords(workspaceContext.workspaceId, entityRecs.map(_.toReference).toSet))
```

Comment:
there was a mismatch caused because `import driver.api._` was changed and inlining `insertInBatches` was the only way I could fix it